config/settings.yml

```
@@ -3,27 +3,10 @@ super_admin_groups: ['sul:requests-super-admin']
 site_admin_groups: ['sul:requests-site-admin']
 
 scan_pilot_groups:
-  - 'stanford:student'
-  - 'stanford:student:postdoc'
-  - 'stanford:student:incoming'
-  - 'stanford:student:contingent'
-  - 'stanford:faculty'
-  - 'stanford:faculty:onleave'
-  - 'stanford:faculty:emeritus'
-  - 'stanford:faculty:otherteaching'
-  - 'stanford:faculty:affiliate'
-  - 'stanford:faculty:slac'
-  - 'stanford:faculty:retired'
-  - 'stanford:faculty:nonactive'
-  - 'stanford:staff'
-  - 'stanford:staff:academic'
-  - 'stanford:staff:onleave'
-  - 'stanford:staff:emeritus'
-  - 'stanford:staff:otherteaching'
-  - 'stanford:staff:affiliate'
-  - 'stanford:affiliate:fellow'
-  - 'stanford:affiliate:visitscholarvs'
-  - 'stanford:affiliate:visitscholarvt'
+  - 'student'
+  - 'faculty'
+  - 'staff'
+  - 'affiliate'
```

The list of groups in #2160 is a little longer:

```suggestion
  - 'faculty'
  - 'fellow'
  - 'graduate'
  - 'postdoctoral'
  - 'staff'
  - 'undergrad'
  - 'visiting scholar'
```

---

thanks